docs/source/interactive.rst

```diff
@@ -5,99 +5,138 @@ Dask-jobqueue is most often used for interactive processing using tools like
+around Jupyter tokens.
+
+
+Start Jupyter
+-------------
+
+When you use Jupyter on your laptop you often just write ``jupyter notebook``
+or ``jupyter lab``.  However, things are a bit different when starting a
+notebook server on a separate machine.  As a first step, the following will
+work:
 
 .. code-block:: bash
 
-   $ ssh -N -L 8888:x.x.x.x:8888 username@hpc_domain
+   jupyter lab --no-browser --ip="*" --port 8888
 
-Now you can go to ``http://localhost:8888`` on your browser to access the
-notebook server.
+Later, once we get SSH tunneling set up, you may want to come back and specify
+a specific IP address or hostname for added security.
 
 
-Viewing the Dask Dashboard
---------------------------
+SSH Tunneling
+-------------
+
+If your personal machine is on the same network as your cluster, then you can
+ignore this step.
+
+If you are on a different network (like your home network), and have to SSH in,
+then it can be difficult to have your local web browser connect to the Jupyter
+server running on the HPC machine.  If your institution doesn't have something
+like `JupyterHub <https://jupyter.org/hub>`_ set up, then the easiest way to
+accomplish this is to use SSH tunneling.
 
-Whether or not you are using dask-jobqueue in Jupyter, IPython or other tools,
-at one point you will want to have access to Dask dashboard. Once you've
-started a cluster and connected a client to it using commands described in
-:ref:`example`), inspecting ``client`` object will give you the Dashboard URL,
-for example ``http://172.16.23.102:8787/status``. The Dask Dashboard may be
-accessible by clicking the link displayed, otherwise, you'll have to use SSH
-tunneling:
+Often a command like the following works:
 
 .. code-block:: bash
 
-    # General syntax
-    $ ssh -fN your-login@scheduler-ip-address -L port-number:localhost:port-number
-    # As applied to this example:
-    $ ssh -fN username@172.16.23.102 -L 8787:localhost:8787
+   ssh -L 8888:login-node-hostname:8888 username@hpc.agency.gov
 
-Now, you can go to ``http://localhost:8787`` on your browser to view the
-dashboard. Note that you can do SSH tunneling for both Jupyter and Dashboard in
-one command.
+Where ...
 
-A good example of using Jupyter along with dask-jobqueue and the Dashboard is
-available below:
+-  ``login-node-hostname`` is the name of the node from which you are
+    running your Jupyter server, designated ``hostname`` below.
 
-.. raw:: html
+    .. code-block:: bash
 
-   <iframe width="560" height="315"
-           src="https://www.youtube.com/embed/nH_AQo8WdKw?rel=0"
-           frameborder="0" allow="autoplay; encrypted-media"
-           allowfullscreen></iframe>
+      username@hostname$ jupyter lab --no-browser --ip="*" --port 8888
 
+    You might also run ``echo $HOSTNAME`` on that machine as well to see the
+    host name.
 
-Dask Dashboard with Jupyter
----------------------------
+-   ``hpc.agency.gov`` is the address that you usually use to
```

Comment:
is it?

Reply:
I've made it more clear that these are placeholders.